### DISCUSSION TITLE
Is there a way to monitor windows services with OTEL?

### DISCUSSION THREAD

**Discussion by majerus1223** [open-telemetry/opentelemetry-collector-contrib#24461] [Q&A]
I am not seeing anything in the documentation for it, just want to make sure I am not missing something. 

Thanks!

https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/receiver/hostmetricsreceiver/README.md

**Comment by Jonas-268** [comment]
Hello, was wondering the same. At the moment it is not possible with OTel, right?

Looks like Influx has a solution for it https://www.influxdata.com/integration/windows-services/ Can you achieve the same with OTel?

**Reply by amccool** [reply]
the prometheus `windows_exporter` https://github.com/prometheus-community/windows_exporter did this really well.   Will that same experience be carried into the otel-collector?
